main_full_par_trafo.g

```diff
@@ -145,238 +149,54 @@ ChiefParallel := function( galoisField,mat,a,b )
-                        ],
-                        UpdateRow,
-                        galoisField,
-                        TaskResult( TaskListClearDown[i][j] ).A,
-                        C[i][k],
-                        TaskResult( TaskListUpdateR[i-1][j][k] ).B,
-                        i
-                    );
-                else
-                    TaskListUpdateR[i][j][k] := ScheduleTask(
-                        [
-                            TaskListClearDown[i][j],
-                            TaskListUpdateR[i-1][j][k],
-                            TaskListUpdateR[i][j-1][k]
-                        ],
-                        UpdateRow,
-                        galoisField,
-                        TaskResult( TaskListClearDown[i][j] ).A,
-                        TaskResult( TaskListUpdateR[i][j-1][k] ).C,
-                        TaskResult( TaskListUpdateR[i-1][j][k] ).B,
-                        i
-                    );
-                fi;
-                
-                
-                
-                #tmp := UpdateRow(  galoisField,A[i][j],C[i][k],
-                #                    B[j][k],i );
-                #C[i][k] := tmp.C;
-                #B[j][k] := tmp.B;
+				UpdateRowInput := UpdateRowParameters(i, j, k, C, TaskListClearDown,
+					TaskListUpdateR, galoisField);
+				TaskListUpdateR[i][j][k] := ScheduleTask(
+					UpdateRowInput[1],
+					UpdateRow,
+					UpdateRowInput[2],
+					UpdateRowInput[3],
+					UpdateRowInput[4],
+					UpdateRowInput[5],
+					UpdateRowInput[6]
+				);
             od;
-            for h in [ 1 .. i ] do
-                
-                if i=1 and j=1 then
-                    TaskListUpdateM[i][j][h] := ScheduleTask(
-                        [
-                            TaskListClearDown[i][j],
-                            TaskListE[h][j]
-                        ],
-                        UpdateRowTrafo,
-                        galoisField,
-                        TaskResult( TaskListClearDown[i][j] ).A,
-                        [],
-                        [],
-                        TaskResult( TaskListE[h][j] ),
-                        i,
-                        h,
-                        j
-                    ); 
-                elif i=1 and j>1 then
-                    TaskListUpdateM[i][j][h] := ScheduleTask(
-                        [
-                            TaskListClearDown[i][j],
-                            TaskListE[h][j],
-                            TaskListUpdateM[i][j-1][h]
-                        ],
-                        UpdateRowTrafo,
-                        galoisField,
-                        TaskResult( TaskListClearDown[i][j] ).A,
-                        TaskResult( TaskListUpdateM[i][j-1][h] ).K,
-                        [],
-                        TaskResult( TaskListE[h][j] ),
-                        i,
-                        h,
-                        j
-                    ); 
 
```

Comment:
Super nitpick:
Could you remove this newline to make the diff look nicer? git displays
```
+            for h in [ 1 .. i ] do
```
as an added line

Reply:
If I remove this line I need to remove all the other lines, but I need these lines to make the code styling better. I really wouldn't want to do this.

Reply:
Ah, I see you also put empty lines in front of for or if blocks in other places? Then scratch what I said. I'm not a big fan of empty lines but my feelings about this are also not too strong.